deleteBuoy.php

```diff
@@ -1,8 +1,16 @@
-<? 
-include('config.php'); 
+<?php 
+include('database.php'); 
 $latitude = (int) $_GET['latitude']; 
 $longitude = (int) $_GET['longitude']; 
-mysql_query("DELETE FROM `buoy` WHERE `latitude` = '$latitude' AND `longitude` = '$longitude' ") ; 
+$sql = "DELETE FROM `buoy` WHERE `latitude` = ? AND `longitude` = ? ";
+$pdo = Database::connect();
+$q = $pdo->prepare($sql);
+try{
+	$q -> execute(array($latitude, $longitude));
+}catch(PDOException $e){
+	die($e->getMessage());
+}
+Database::disconnect();
 echo (mysql_affected_rows()) ? "Row deleted.<br /> " : "Nothing deleted.<br /> "; 
```

Comment:
Did you figure out if `mysql_affected_rows()` works when using PDO (rather than the `mysql_` functions)?